tmp/tmp.d.ts

```diff
@@ -0,0 +1,33 @@
+﻿// Type definitions for tmp v0.0.24
+// Project: https://github.com/raszi/node-tmp

```

Comment:
Prefer NPM link : https://www.npmjs.com/package/tmp 

Other than that great job!